media/js/firefox/australis/browser-tour.js

```diff
@@ -504,8 +524,13 @@ if (typeof Mozilla == 'undefined') {
             buttons,
             options
         );
-
-        gaTrack(['_trackEvent', 'Tour Interaction', 'Later', 'The new Forget Button']);
+        
+        window.dataLayer = window.dataLayer || [];
```

Comment:
This particular one happens after user interaction, but can also happen via other events, such as page visibility changes. It does not show automatically on page load however.